src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,181 @@
+package seedu.duke;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.io.FileNotFoundException;
+
+/**
+ * The main class.
+ */
+public class Duke extends Application {
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
+    private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
+
+    private static Storage storage;
+    private static TaskList tasks;
+    private static Ui ui;
+    private static boolean isExit;
+
+    /** Constructor.*/
+    public Duke(String filepath) {
+        ui = new Ui();
+        storage = new Storage(filepath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (FileNotFoundException e) {
+            ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+
+    /**
+     * While user has yet to input the exit command, each user-inputted command is parsed and executed.
+     */
+    public void run() {
+        ui.intro();
+        boolean isExit = false;
+        while (!isExit) {
+            try {
+                String fullCommand = ui.readCommand();
+                Command c = Parser.parse(fullCommand);
+                c.execute(tasks, ui, storage);
+                isExit = c.isExit();
+            } catch (DukeException e) {
+                ui.showError(e.getMessage());
+            }
+        }
+    }
+
+    /**
+     * Runs process.
+     * @param args list of String arguments.
+     */
+    public static void main(String[] args) {
+        new Duke("data/tasks.txt").run();
+    }
+
+    @Override
+    public void start(Stage stage) {
+        //Step 1. Formatting the window to look as expected.
```

Comment:
I don't think you need these codes anymore since Duke will be launched from Main instead.